defaults/config.js

```diff
@@ -35,6 +35,19 @@ module.exports = {
 	bind: undefined,
 
 	//
+	// Enable or disable pid file creation
+	//
+	// string values specify the file path to use
+	// boolean values result in the default file path being used
+	//
+	// paths are relative to the 'HOME' folder
+	//
```

Comment:
For consistency with other paths in config (ex: [ssl key](https://github.com/erming/shout/blob/5bfc39bbc9972a68b5a287ed5d7e7e28beb32995/src/server.js#L31)), this should not be relative to HOME folder (+ it's a limitation).